client/app/iplb/configuration/iplb-configuration.html

```diff
@@ -2,29 +2,64 @@
     <cui-page-content-title text="'iplb_configuration_title' | translate"></cui-page-content-title>
     <p data-translate="iplb_configuration_intro"></p>
 
-    <button type="button"
-        class="oui-button oui-button_secondary"
-        data-translate="iplb_configuration_action_apply"
-        data-ng-click="ctrl.applyChanges()"
-        data-ng-disabled="!ctrl.selectedZones.length"></button>
+    <div
+        class="text-center"
+        data-ng-if="$ctrl.zones.loading">
+        <oui-spinner></oui-spinner>
+    </div>
 
-    <oui-spinner data-ng-show="ctrl.zones.loading"></oui-spinner>
-    <oui-table data-ng-hide="ctrl.zones.loading"
-        rows="ctrl.zones.data"
-        on-selection-change="ctrl.onSelectionChange($selection)">
-        <column title="'iplb_configuration_datacenter' | translate" property="name" sortable="asc"></column>
-        <column title="'iplb_configuration_progress' | translate"
-            sortable
-            class="oui-table__cell_component">
-            <oui-progress data-compact data-ng-show="$row.task">
-                <oui-progress-bar data-type="{{ ctrl.ServiceHelper.constructor.getTaskProgressType($row.task.status) }}" data-value="$row.task.progress">
-                </oui-progress-bar>
+    <oui-datagrid
+        data-ng-if="!$ctrl.zones.loading"
+        data-rows="$ctrl.zones.data"
+        data-selectable-rows>
+        <extra-top>
+            <oui-button
+                class="mb-3"
+                data-variant="secondary"
+                data-text="{{:: 'iplb_configuration_action_apply' | translate }}"
+                data-on-click="$ctrl.applyChanges($selectedRows)"
+                data-disabled="!$selectedRows.length"></oui-button>
+        </extra-top>
+        <oui-column
+            data-title="'iplb_configuration_datacenter' | translate"
+            data-property="name"
+            data-sortable="asc"></oui-column>
+        <oui-column
+            data-title="'iplb_configuration_progress' | translate"
+            data-sortable>
+            <oui-progress
+                data-compact
+                data-ng-if="$row.task">
+                <oui-progress-bar
+                    data-type="{{ $ctrl.ServiceHelper.constructor.getTaskProgressType($row.task.status) }}"
```

Comment:
No `::`

Reply:
Well it's a progress bar, I do hope the status changes with the progression :)

Reply:
No, in fact the progress bar type property has one-time binding, so `::` is possible I guess.
See https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-progress/src/bar/progress-bar.controller.js#L13.